pkgs/top-level/aliases.nix

```diff
@@ -320,6 +320,23 @@ mapAliases (rec {
   libva-full = libva;
   libva1-full = libva1;
 
+  # cross aliases
+
+  arm-frc-linux-gnueabi-eglibc = pkgsCross.arm-frc-linux-gnueabi.eglibc;
+  arm-frc-linux-gnueabi-linux-api-headers = pkgsCross.arm-frc-linux-gnueabi.buildPackages.linuxHeaders;
+  arm-frc-linux-gnueabi-binutils = pkgsCross.arm-frc-linux-gnueabi.buildPackages.binutils;
+  arm-frc-linux-gnueabi-gcc = pkgsCross.arm-frc-linux-gnueabi.buildPackages.gcc;
+
+  gcc-arm-embedded-4_8 = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc48;
+  gcc-arm-embedded-4_9 = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc49;
+  gcc-arm-embedded-5 = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc5;
+  gcc-arm-embedded-6 = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc6;
+  gcc-arm-embedded = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc;
+
+  avrgcclibc = throw "avrgcclibs are now separate packages, install avrbinutils, avrgcc and avrlibc";
+  avrbinutils = pkgsCross.avr-atmel.buildPackages.binutils;
+  avrgcc = pkgsCross.avr-atmel.buildPackages.gcc;
```

Comment:
Is this going to contain wrapped versions of the toolchains? I doubt that's going to work for the 8-bit micros since all the hardening stuff etc. is irrelevant and probably harmful.

Reply:
Yeah I wonder how to handle that. There is a gcc-unwrapped correct?

Reply:
Actually, maybe we can disable hardening for certain target platforms?